cmd/bridge/main.go

```diff
@@ -591,9 +592,9 @@ func main() {
 		Handler: srv.HTTPHandler(),
 		// Disable HTTP/2, which breaks WebSockets.
 		TLSNextProto: make(map[string]func(*http.Server, *tls.Conn, http.Handler)),
-		TLSConfig: &tls.Config{
+		TLSConfig: oscrypto.SecureTLSConfig(&tls.Config{
 			CipherSuites: crypto.DefaultCiphers(),
```

Comment:
I think we can remove `crypto.DefaultCiphers()` now and associated module since we have library-go.